samples/exit-handler-email/send-email.py

```diff
@@ -14,10 +14,9 @@
 
 from kfp import dsl
 import kfp
-from kubernetes import client as k8s_client
 from kfp.components import func_to_container_op
 from kfp import onprem
-import os
+from kfp_tekton.k8s_client_helper import env_from_secret
```

Comment:
I saw you defined the env_from_secret here, but I didn't see any caller in this script.

Reply:
it's being used at line 51-55

Reply:
Yeah, that's stange, why the 51-55 are not shown in the diff.